dist/main.css

```diff
@@ -399,7 +399,7 @@ body {
   max-width: 1400px;
   margin: 0 auto;
   font-family: 'Poppins', sans-serif;
-  background-color: #F2F2F2; }
+  background-color: #FFFFFF; }
```

Comment:
Remove dist folder. Add dist folder to .gitignore

Reply:
completed